src/web/views/send-code.njk

```diff
@@ -19,14 +19,17 @@
                 value: "text",
                 text: text,
                 checked : claim.channelForCode === "text",
-                conditional: {
-                html: eddHtml
-            }
+                hint: {
+                    html: claim.phoneNumber + " <a href=\"/phone-number\" id=\"change-text\">" + change + "</a>"
```

Comment:
Shouldn't this use the value from the translations file? Same below too?

Reply:
it is!

Reply:
Oh, yes, missed that.